packages/react-jsx-highcharts/src/components/WithHighcharts/index.js

```diff
@@ -16,3 +16,9 @@ export default function withHighcharts(Component, Highcharts) {
     );
   }
 }
+
+export const WithHighcharts = ({ Highcharts, children }) => (
```

Comment:
Should we change the default export above to render

```jsx
export default function withHighcharts(Component, Highcharts) {
  // Maybe add a deprecate notice here? Although I feel this HOC component still has some value
  return props => (
      <WithHighcharts Highcharts={Highcharts}>
        <Component {...props} />
      </WithHighcharts>
    )
  }
}
```

Reply:
Done.